fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/matching/structure/QuantifierMatchers.java

```diff
@@ -78,6 +79,13 @@ public static BindingMatcher<Quantifier> anyQuantifierOverRef(@Nonnull final Bin
         return ofTypeRangingOverRef(Quantifier.class, downstream);
     }
 
+    @Nonnull
+    public static BindingMatcher<Quantifier> anyQuantifierWithAlias(@Nonnull final BindingMatcher<CorrelationIdentifier> alias) {
```

Comment:
Matchers on aliases will only work if the debugger is installed. So if you have a test scenario where you need this, it's fine, it won't work if `DebuggerWithSymbolTables` is not present as the alias is not stable.